<?xml version="1.0"?>
<launch>
   <node name="sirona_sm_node" pkg="sirona" type="sirona_sm_node" output="screen">
      <param name="pub_re_target_topic"     type="string" value="rona/plan/target"          />
      <param name="pub_re_addob_topic"      type="string" value="rona/plan/add_obstacle"    />
      <param name="pub_re_rmob_topic"       type="string" value="rona/plan/rm_obstacle"    />
      <param name="pub_replan_topic"        type="string" value="rona/plan/replan"          />
      <param name="pub_path_topic"          type="string" value="rona/move/path"            />
      <param name="pub_state_topic"         type="string" value="rona/state"                />
      <param name="pub_move_ctrl_topic"     type="string" value="rona/move/ctrl"            />
      <param name="sub_target_topic"        type="string" value="/move_base_simple/goal"    />
      <param name="sub_addob_topic"         type="string" value="rona/add_obstacle"         />
      <param name="sub_rmob_topic"          type="string" value="rona/add_obstacle"         />
      <param name="sub_path_topic"          type="string" value="rona/plan/path"            />
      <param name="sub_state_topic"         type="string" value="rona/move/state"           />
      <param name="robot_frame"             type="string" value="base_footprint"            />
      <param name="map_frame"               type="string" value="map"                       />
      <param name="obstacle_block_time"     type="double" value="2.0"    />
      <param name="ob_react_poly_0_x"       type="double" value="0.0"    />
      <param name="ob_react_poly_0_y"       type="double" value="0.3"    />
      <param name="ob_react_poly_1_x"       type="double" value="0.0"    />
      <param name="ob_react_poly_1_y"       type="double" value="-0.3"    />
      <param name="ob_react_poly_2_x"       type="double" value="1.2"    />
      <param name="ob_react_poly_2_y"       type="double" value="-0.5"    />
      <param name="ob_react_poly_3_x"       type="double" value="1.2"    />
      <param name="ob_react_poly_3_y"       type="double" value="0.5"    />
   </node>
</launch>